x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -234,6 +246,19 @@ export const showView: (state: EndpointState) => 'policy_response' | 'details' =
   }
 );
 
+/**
+ * Returns the selected endpoint's elastic agent Id
+ * used for fetching endpoint actions log
+ */
+export const selectedAgent = (state: Immutable<EndpointState>) => {
```

Comment:
Suggestion: add explicit return type to this function's signature

Reply:
2dd22ed92b3c54da9bdcce467dbe2d7e87097e4d